spec/spec_helper.rb

```diff
@@ -38,4 +38,19 @@ def change_address
   def sign_tx(tx, _prevtxs = [])
     tx
   end
+
+  def collect_uncolored_outputs(amount)
+    utxos = list_unspent
+
+    utxos.inject([0, []]) do |sum, output|
+      next sum if output[:color_id]
+
+      new_sum = sum[0] + output[:amount]
+      new_outputs = sum[1] << output
+      return [new_sum, new_outputs] if new_sum >= amount
+
+      [new_sum, new_outputs]
+    end
+    raise Glueby::Contract::Errors::InsufficientFunds
+  end
```

Comment:
Why do you need this method?

Reply:
Because `TestWallet` was created from this file, So can't use `Glueby::Internal::Wallet.collect_uncolored_outputs`.

Reply:
If `TestInternalWallet` inherits from `Glueby::Internal::Wallet`, you don't have to write duplicate code.